client/src/components/cart/CartTable.js

```diff
@@ -0,0 +1,187 @@
+import { useContext } from "react";
+import styled from "styled-components";
+import Payment from "../forms/Payment";
+import { Link } from "react-router-dom";
+import { CartContext } from "../context/Context";
+function CartTable() {
+  const [cart, setCart] = useContext(CartContext);
+  const toggleItemQuantity = (itemName, subtract) => {
+    let newState;
+    if (subtract) {
```

Comment:
Here, I'd rather be explicit like: `if (action == 'subtract') {}`

Reply:
Will just lift this code and implement, thanks. 👍